docs/deployed-contracts/index.md

```diff
@@ -47,11 +52,36 @@ pagination_next: deployed-contracts/holesky
 - Gate Seal: [`0x79243345eDbe01A7E42EDfF5900156700d22611c`](https://etherscan.io/address/0x79243345eDbe01A7E42EDfF5900156700d22611c)
 
+## Staking modules
+
+- Curated (Node Operators registry): [`0x55032650b14df07b85bF18A3a3eC8E0Af2e028d5`](https://etherscan.io/address/0x55032650b14df07b85bF18A3a3eC8E0Af2e028d5) (proxy)
+- Curated (Node Operators registry): [`0x8538930c385C0438A357d2c25CB3eAD95Ab6D8ed`](https://etherscan.io/address/0x8538930c385C0438A357d2c25CB3eAD95Ab6D8ed) (impl)
+- ([proposed](https://research.lido.fi/t/staking-router-community-staking-module-upgrade-announcement/8612/2)) Curated (Node Operators registry): [`0x1770044a38402e3cfca2fcfa0c84a093c9b42135`](https://etherscan.io/address/0x1770044a38402e3cfca2fcfa0c84a093c9b42135) (impl)
+- Simple DVT: [`0xaE7B191A31f627b4eB1d4DaC64eaB9976995b433`](https://etherscan.io/address/0xaE7B191A31f627b4eB1d4DaC64eaB9976995b433) (proxy)
+- Simple DVT: [`0x8538930c385C0438A357d2c25CB3eAD95Ab6D8ed`](https://etherscan.io/address/0x8538930c385C0438A357d2c25CB3eAD95Ab6D8ed) (impl)
+- ([proposed](https://research.lido.fi/t/staking-router-community-staking-module-upgrade-announcement/8612/2)) Simple DVT: [`0x1770044a38402e3cfca2fcfa0c84a093c9b42135`](https://etherscan.io/address/0x1770044a38402e3cfca2fcfa0c84a093c9b42135) (impl)
+
+### Community Staking Module
+
+- ([proposed](https://research.lido.fi/t/community-staking-module/5917/67)) CSModule: [`0xdA7dE2ECdDfccC6c3AF10108Db212ACBBf9EA83F`](https://etherscan.io/address/0xdA7dE2ECdDfccC6c3AF10108Db212ACBBf9EA83F) (proxy)
+- ([proposed](https://research.lido.fi/t/community-staking-module/5917/67)) CSModule: [`0x8daea53b17a629918cdfab785c5c74077c1d895b`](https://etherscan.io/address/0x8daea53b17a629918cdfab785c5c74077c1d895b) (impl)
+- ([proposed](https://research.lido.fi/t/community-staking-module/5917/67)) CSAccounting: [`0x4d72BFF1BeaC69925F8Bd12526a39BAAb069e5Da`](https://etherscan.io/address/0x4d72BFF1BeaC69925F8Bd12526a39BAAb069e5Da) (proxy)
+- ([proposed](https://research.lido.fi/t/community-staking-module/5917/67)) CSAccounting: [`0x71FCD2a6F38B644641B0F46c345Ea03Daabf2758`](https://etherscan.io/address/0x71FCD2a6F38B644641B0F46c345Ea03Daabf2758) (impl)
+- ([proposed](https://research.lido.fi/t/community-staking-module/5917/67)) CSFeeDistributor: [`0xD99CC66fEC647E68294C6477B40fC7E0F6F618D0`](https://etherscan.io/address/0xD99CC66fEC647E68294C6477B40fC7E0F6F618D0) (proxy)
+- ([proposed](https://research.lido.fi/t/community-staking-module/5917/67)) CSFeeDistributor: [`0x17Fc610ecbbAc3f99751b3B2aAc1bA2b22E444f0`](https://etherscan.io/address/0x17Fc610ecbbAc3f99751b3B2aAc1bA2b22E444f0) (impl)
+- ([proposed](https://research.lido.fi/t/community-staking-module/5917/67)) CSFeeOracle: [`0x4D4074628678Bd302921c20573EEa1ed38DdF7FB`](https://etherscan.io/address/0x4D4074628678Bd302921c20573EEa1ed38DdF7FB) (proxy)
```

Comment:
Lets format CSFeeOracle proxy, implementation and consensus in the same way as all other protocol oracles
- CSFeeOracle:
   - CSFeeOracle: (proxy)
   - CSFeeOracle: (impl)
   - HashConsensus: